wordpress-mobile/WordPress-iOS
# Posts and Pages search: Update to Muriel color scheme

This PR fixes the color and some design details in posts and pages search.

To test:

- In the posts section, perform a search and check if the design matches the blueprint of #11646
- In the pages section, perform a search and check if the design matches the blueprint of #11646

| Before  | After |
| ------------- | ------------- |
|  <img src="https://user-images.githubusercontent.com/7040243/61254088-bc8f6780-a739-11e9-8236-f1eb4eebfcee.png" width="300px" />  | <img src="https://user-images.githubusercontent.com/7040243/61254118-d3ce5500-a739-11e9-8081-1b2702824b56.png" width="300px" /> |
